### ISSUE TITLE
Add warnings for bad audio values

### ISSUE THREAD

**Issue by JulianGro** [vircadia/vircadia#925]
Currently we have a couple of problems with the audio system being high latency or unreliable.
In Dales case he had an extremely high `Output Read` latency. (over 3000ms)
In Mr.Squarepegs case he had extremely high Jitter values, both upstream and downstream. (over 1000ms)
In both these cases there was something pretty obviously wrong with the values and the users had no idea.
I feel like Interface should warn its users or at least write to log when something starts getting out of sane values like that.

Without knowing how every system and network behaves, I would suggest the following values:

Input Read = >200ms
Input Ring = >50ms  (my understanding is that that is the jitter between the local systems audio server and Interface)
Network (up) = >400ms
Mixer Ring = >200ms
Network (down) = >400ms
Output Ring = >200ms
Output Read = >400ms

Upstream Jitter = not sure what this is

Mixer (upstream) Jitter = >350ms
Mixer (upstream) Packet loss = >0.05%

Client (downstream) Jitter = >350ms
Client (downstream) Packet loss = >0.05%

**Comment by daleglass** [comment]
Dev meeting: In my case PipeWire seems to have fixed latency issues, but this would still be useful

**Comment by daleglass** [comment]
Still a good idea, though my original issue is gone
